package.json

```diff
@@ -4,7 +4,8 @@
   "author": "Tobias Koppers @sokra",
   "description": "less loader module for webpack",
   "scripts": {
-    "test": "node --no-deprecation node_modules/.bin/_mocha -R spec",
+    "test": "npm run travis:test",
+    "travis:test": "node --no-deprecation node_modules/.bin/_mocha -R spec",
```

Comment:
- Why is there an extra command for travis:test? We should just define `test`
- I know that this has been there before, but why `--no-deprecation`?